shared/src/main/java/com/couchbase/lite/ArrayExpression.java

```diff
@@ -41,8 +43,13 @@ private ArrayExpression() {
      *
      * @param variable The variable expression.
      * @return An In object
+     * @throws IllegalArgumentException when variable is null.
      */
-    public static ArrayExpressionIn any(VariableExpression variable) {
+    public static ArrayExpressionIn any(@NonNull VariableExpression variable) {
+
+        if(variable == null) {
```

Comment:
Please place a space b/t if and (.

if (variable == null)

Please check others as well.

Reply:
Please also remove the extra empty line. Same for the other methods.